Resources/doc/definitions/graphiql/custom-http-headers.md

```diff
@@ -0,0 +1,48 @@
+Custom HTTP headers
+==============
+
+GraphiQL, provided by this bundle, sends the following default headers on each request:
+
+```js
+headers = {
+  "Accept": "application/json",
+  "Content-Type": "application/json"
+};
+```
+
+Headers sent by GraphiQL can be modified. 
+For example, let's assume an `access-token` header is required in development.
+The header can be added the following way:
+
+1. Override the default GraphiQL template using parameter:
+
+```yml
+# app/config/config_dev.yml
+parameters:
```

Comment:
this can be replaced by the bundle configuration:
```yaml
overblog_graphql:
    templates:
        graphiql: AppBundle:GraphiQL:index.html.twig
```